Assets/Scripts/UI/InGameUI/MenuTop/CurrencyDisplay.cs

```diff
@@ -13,7 +13,8 @@
 
 public class CurrencyDisplay : MonoBehaviour
 {
-    public Text text;
+    [SerializeField]
+    private Text text;
```

Comment:
Does this even need serialized, I realize adding that keeps closer to the previous behavior, but since we do so little work in editor, I'm not sure it even needs serialized (serialization is the editor is the only benefit I can think of for this).